0025-validators.md

```diff
@@ -36,24 +36,36 @@ We also expect the Testnet process, wherein we will stand up a separate network
+  * `unstake_validator_v1`: validator pubkey, owner pubkey, stake, release height: this starts the unlock process for the stake, returning it to the owner's account after some withdrawal period.  The validator is immediately unable to join block production groups
+  * `validator_heartbeat_v1`: validator pubkey, height, version: this provides the participants in the chain information about an individual validator's version and online status
   
-#### Chain Variables:
-  * Activation Threshold: minimum number of staked Validators to start using them for elections. After the Activation Threshold is met, the next election will produce a list of Validators rather than draw from the full Hotspot pool. After the Threshold is reached and the chain variable is active, we will no longer go back to selecting from the Hotspot pool.  The initial proposed value here is 100 Validators.
-  * Activation Delay: the number of blocks after which (once the above threshold has been met) the group will start moving to the Validator pool.  Unstaking to below the activation threshold during the delay period will cancel activation.
-  * Minimum Stake: the minimum number of HNT that need to be staked before the Validator can be created and considered for Block Producer elections.  The proposed initial value for the minimum stake is 10,000 HNT.
-  * Cooldown Period: The number of blocks after unstaking when the stake will be returned to the owner's account.  Proposed initial value is 250,000 blocks, approximately 5 months.
+#### Primary Chain Variables
 
+These chain variables are used to govern the qualifications of validators on the Helium blockchain:
+
+  * Validator Version (`validator_version`): This version variable will be used to control the roll out of validators. Initially this value will be `1` in order to allow staking transactions to be accepted by the blockchain in advance of activating validators. When this version is updated to `2`, validators will be selected as Block Producers and this HIP is officially launched. We propose that the transition from version `1` to version `2` be the _longer_ of 2 weeks or the period needed to have at least 100 validators staked and producing heartbeats.
+  * Stake (`validator_minimum_stake`): the amount of HNT that needs to be staked for the Validator to be considered for Block Producer elections. This value will initially be fixed at **10,000 HNT** and there will be no benefit of over or under staking.
+  * Stake Withdrawal Period (`stake_withdrawal_cooldown`): This will be the minimum number of blocks after unstaking when the stake will be returned to the owner's account. This will be set to **125,000 blocks** at the launch of staking (at `validator_version` = `1`) and a block height deadline needs to be provided by the validator owner at unstaking. An additional variable (`stake_withdrawal_max`) will also be available to prevent validator operators to set their withdrawal deadline too far in the future.
```

Comment:
```suggestion
  * Stake Withdrawal Period (`stake_withdrawal_cooldown`): This will be the minimum number of blocks after unstaking when the stake will be returned to the owner's account. This will be set to **250,000 blocks** at the launch of staking (at `validator_version` = `1`) and a block height deadline needs to be provided by the validator owner at unstaking. An additional variable (`stake_withdrawal_max`) will also be available to prevent validator operators to set their withdrawal deadline too far in the future.
```